hbw/app/javascript/packs/hbw/components/form/checkbox.js.jsx

```diff
@@ -41,6 +42,11 @@ modulejs.define('HBWFormCheckbox', ['React'], (React) => {
 
       const labelCSS = cx('hbw-checkbox-label', this.props.params.label_css);
 
+      const checkedIcon = this.props.params.icon?.checked || ['far', 'check-square'];
+      const uncheckedIcon = this.props.params.icon?.unchecked || ['far', 'square'];
```

Comment:
`this.props.` is redundant here, it's already destructured.


Also, it could be expressed like this
```
const { 
  checked: checkedIcon = ['far', 'check-square'],
  unchecked: uncheckedIcon = ['far', 'square'],
} = params.icon;
```
but it's up to you)